src/Compute/Compute/help/New-AzVM.md

```diff
@@ -57,11 +57,18 @@ New-AzVM [[-ResourceGroupName] <String>] [[-Location] <String>] [-EdgeZone <Stri
 
 ## DESCRIPTION
 The **New-AzVM** cmdlet creates a virtual machine in Azure.
-This cmdlet takes a virtual machine object as input.
-Use the New-AzVMConfig cmdlet to create a virtual machine object.
-The **New-AzVM** cmdlet will create a new storage account for boot diagnostics if one does not already exist. 
-Other cmdlets can be used to configure the virtual machine, such as Set-AzVMOperatingSystem, Set-AzVMSourceImage, Add-AzVMNetworkInterface, and Set-AzVMOSDisk.
-The `SimpleParameterSet` provides a convenient method to create a VM by making common VM creation arguments optional.
+This cmdlet takes a virtual machine object as input. The **New-AzVM** cmdlet will create a new storage account for boot diagnostics if one does not already exist. <br> <br>
+Use the **[New-AzVMConfig](https://docs.microsoft.com/en-us/powershell/module/az.compute/new-azvmconfig)** cmdlet to create a virtual machine object. <br> 
+Then use the following cmdlets to set different properties of the virtual machine object:
+- **[Add-AzVMNetworkInterface](https://learn.microsoft.com/en-us/powershell/module/az.compute/add-azvmnetworkinterface)** to set the network profile.<br>
+- **[Set-AzVMOperatingSystem](https://learn.microsoft.com/en-us/powershell/module/az.compute/set-azvmoperatingsystem)** to set the OS profile. <br>
+- **[Set-AzVMSourceImage](https://learn.microsoft.com/en-us/powershell/module/az.compute/set-azvmsourceimage)** to set the source image.<br>
+- **[Set-AzVMOSDisk](https://learn.microsoft.com/en-us/powershell/module/az.compute/set-azvmosdisk)** to set the OS disk(storage profile).<br>
+- **[Get-AzComputeResourceSku](https://docs.microsoft.com/en-us/powershell/module/az.compute/get-azcomputeresourcesku)** can also be used to find out available virtual machine sizes for your subscription and region.<br>
+<br>
+The `SimpleParameterSet` provides a convenient method to create a VM by making common VM creation arguments optional.<br>
+<br>
+See [Quickstart: Create a Windows virtual machine in Azure with PowerShell](https://learn.microsoft.com/en-us/azure/virtual-machines/windows/quick-create-powershell) for tutorial. <br>
```

Comment:
this line, the hyper link is not working for some reason. Do have any idea why?
The above lines work.
See here: https://github.com/Azure/azure-powershell/blob/514716982d8f0c82b97d4235f392ac356204368d/src/Compute/Compute/help/New-AzVM.md